sky/packages/sky/lib/src/material/scrollbar_painter.dart

```diff
@@ -0,0 +1,102 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:flutter/animation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+const double _kMinScrollbarThumbLength = 18.0;
+const double _kScrollbarThumbGirth = 6.0;
+const Duration _kScrollbarThumbFadeDuration = const Duration(milliseconds: 300);
+
+class MaterialScrollbarPainter extends ScrollingListPainter {
+
+  double get opacity => _opacity;
+  double _opacity = 0.0;
+  void set opacity (double value) {
+    assert(value != null);
+    assert(value >= 0.0 && value <= 1.0);
+    if (_opacity == value)
+      return;
+    _opacity = value;
+    renderer?.markNeedsPaint();
+  }
+
+  int get alpha => (opacity * 0xFF).round();
+
+  // TODO(hansmuller): thumb color should come from ThemeData.
+  Color get thumbColor => const Color(0xFF9E9E9E);
+
+  void paintThumb(PaintingContext context, Rect thumbBounds) {
+    Paint paint = new Paint()
+      ..color = thumbColor.withAlpha(alpha)
+      ..style = ui.PaintingStyle.fill;
```

Comment:
The Paint class (sky/engine/core/painting/Paint.dart) doesn't seem to define a default PaintingStyle.  Should it? Or maybe I'm misunderstanding how the Paint constructor ends up working?